apps/web-giddh/src/app/shared/tax-sidebar/tax-sidebar.component.html

```diff
@@ -229,8 +229,60 @@
                             <span>{{ localeData?.filing_status }}</span>
                         </a>
                     </li>
+                    <li *ngIf="showSalesTaxMenus && isUSCompany">
```

Comment:
add ng-container and add this if condition there and remove condition from all list elements

Reply:
Done